kayenta-gcs/src/main/java/com/netflix/kayenta/gcs/storage/GcsStorageService.java

```diff
@@ -100,6 +100,12 @@ public void ensureBucketExists(String accountName) {
           log.error("Could not create bucket {} in project {}: {}", bucketName, projectName, e2);
           throw new IllegalArgumentException(e2);
         }
+      } else if (e.getStatusCode() == 403) {
+        log.error(
+            "Account does not have permissions to get bucket metadata {}. Please see the FAQ for details: https://github.com/spinnaker/kayenta/blob/master/docs/faq.md: {}",
```

Comment:
Looks like you can deep-link directly to the new faq entry (once it's merged). Likely just need to add `#my-gcs-account-does-not-have-access-to-get-bucket-metadata`.

Reply:
Good catch.